app/controllers/wpcc/your_details_controller.rb

```diff
@@ -8,11 +10,13 @@ def new
 
     def create
       @your_details_form = Wpcc::YourDetailsForm.new(your_details_form_params)
+      @your_details_form = present(@your_details_form)
```

Comment:
either all on one line as @margOnline suggests, or perhaps rename the variables.